src/Controllers/AccountController.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MvcMovie.Repositories.Interfaces;
+using MvcMovie.Repositories.Database;
+using MvcMovie.Models.Views;
+using MvcMovie.Models.Database;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Authentication;
+
+namespace MvcMovie.Controllers
+{
+    [Route("[controller]")]
+    public class AccountController : Controller
+    {
+        private readonly UserManager<User> _userManager;
+        private readonly SignInManager<User> _signInManager;
+
+        public AccountController(UserManager<User> userManager, SignInManager<User> signInManager)
+        {
+            this._userManager = userManager;
+            this._signInManager = signInManager;
+        }
+        
+        public SignInManager<User> SignInManager { get { return this._signInManager; } }
+        public UserManager<User> UserManager { get { return this._userManager; } }
+
+        [HttpGet("Register")]
+        [AllowAnonymous]
+        public IActionResult Register() => View();
+
+        [HttpPost("Register")]
+        [AllowAnonymous]
+        public async Task<IActionResult> Register(UserViewModel uvm)
+        {
+            if (ModelState.IsValid)
+            {
+                var user = new User { UserName = uvm.UserName, Email = uvm.Email };
```

Comment:
Be careful! What happens if the user already exists? Please, create an existent user and check if the error appears in the view.

Reply:
It doesn't show the error in the view because there is no asp-validation-for in the Register.cshtml, but it also doesn't create a new user in the database. Later, in the Validation branch, the error appears in the view